### ISSUE TITLE
Projeto

### ISSUE THREAD

**Issue by lucianosousa** [doadordecodigo/doadordecodigo#2]
Cada projeto(solicitação de) terá uma descrição e estará relacionado à um usuário, a pessoa do projeto social, por exemplo.

Usuários podem 'aceitar' o projeto e começarem a participar dele.
O projeto precisa ter um 'manager', uma figura central que vai configurar ele e também avaliar PR dos usuários iniciantes.
Inicialmente, o projeto pode começar só com um manager, mas no futuro a idéia é ter mais de um manager por projeto

**Comment by gustavokloh** [comment]
O `Idealizador` e o `Manager` são usuários membros do `Projeto`? Assim o `Usuário` terá essas flags de alguma forma.

Entendi corretamente?

**Comment by lucianosousa** [comment]
Sim. Acho que seria interessante utilizar o pundit para isso, melhor que tratar na mão todos esses casos.

**Comment by lucianosousa** [comment]
Além desses dois papéis, temos o papel também do desenvolvedor, que possui outros privilégios nessa hierarquia.

**Comment by lucianosousa** [comment]
@gustavokloh você está fazendo isso ou eu posso pegar?

**Comment by gustavokloh** [comment]
@lucianosousa pode pegar ;)
